devops/make/common-go.mk

```diff
@@ -16,7 +15,7 @@ deps:: deps-go
 deps-circle:: deps-circle-go deps
 lint:: lint-go
 test:: lint-go test-go-tparse
-test-circle:: test test-coveralls
+test-circle:: test-coverage-go test
```

Comment:
Making this change to devops/make at the project level will cause a merge conflict down the line. Instead, this should be updated in https://github.com/pantheon-systems/common_makefiles/ and changes applied here per the common make documentation.

Reminder, if making changes to common make, merge the PR, do not use squash or rebase.

Reply:
Done https://github.com/pantheon-systems/common_makefiles/pull/398

Reply:
@pwtyler could you please rereview that PR?